app/models/concerns/project_media_getters.rb

```diff
@@ -152,6 +152,15 @@ def get_title
     title.to_s
   end
 
+  def media_slug
+    title = ''
+    if self.user&.login == 'smooch'
+      analysis = self.analysis
+      title = [analysis['file_title'], analysis['title']].compact_blank.first
+    end
+    title
+  end
+
```

Comment:
Suggestion: Reuse this method in https://github.com/meedan/check-api/blob/develop/app/models/concerns/project_media_getters.rb#L147-L153 instead of duplicating the logic.

Reply:
Already `ProjectMedia.title` call this method and this one override original title by FactCheck title or` ClaimDescription` description but in this case we just need an analysis.title for claim or links or analysis.file_title for media items